pkg/ebpf/btf.go

```diff
@@ -174,33 +214,33 @@ func (b *orderedBTFLoader) embeddedPath() (string, error) {
 
 var kernelVersionPatterns = []struct {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏This doesn't need SUSE/SLES?

Reply:
Nope, these are patterns which let us infer possible platforms from the just the kernel version. SUSE/SLES doesn't have any identifying strings unfortunately.

Reply:
Cool - thanks!